src/Controller/EnMarche/DonationController.php

```diff
@@ -133,4 +137,44 @@ public function resultAction(Request $request, Donation $donation)
             'retry_url' => $retryUrl,
         ]);
     }
+
+    /**
+     * @Route(
+     *     "/mensuel/cancel",
+     *     defaults={"_enable_campaign_silence"=true},
+     *     name="donation_subscription_cancel"
+     * )
+     * @Method("GET")
```

Comment:
why only `GET`?
Why do you not have the confirmation form: `Etes-vous sûr de vouloir annuler votre subcription mensuele? oui / non` with check of ** CSRF ** token??

Reply:
Totaly forgot thx